ops/training/_data.tf

```diff
@@ -51,12 +51,6 @@ data "azurerm_key_vault_secret" "sr_db_jdbc" {
   key_vault_id = data.azurerm_key_vault.global.id
 }
 
-
-data "azurerm_key_vault_secret" "slack_notify_webhook_url" {
```

Comment:
do we need a follow up ticket to remove secrets from Azure, or are they still used?

Reply:
Good call, will do